testing/iprutils/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Maintainer: Breno Leitao <breno.leitao@gmail.com>
+pkgname=iprutils
+pkgver=2.4.14
+pkgrel=0
+pkgdesc="user space tools required by the ipr device driver."
+url="https://github.com/bjking1/iprutils"
+arch="all"
+license="CPL"
+depends="automake libtool m4 autoconf ncurses-dev zlib-dev"
+makedepends=""
+install=
+subpackages="$pkgname-doc"
+_pkgver=$(echo $pkgver | sed 's/\./-/g')
+source="$pkgname-$pkgver.tar.gz::https://github.com/bjking1/$pkgname/archive/rel-$_pkgver.tar.gz"
+
+_builddir="$srcdir"/iprutils-rel-$_pkgver
```

Comment:
This variable should be `builddir`. This got renamed recently, when we added default_prepare to use it.